config.selenium-standalone-chrome.yaml

```diff
@@ -4,6 +4,16 @@
 #
 # This file comes from https://github.com/drud/ddev-selenium-standalone-chrome
 #
+hooks:
+  # Make sure that the standard mkcert CA keys are readable
+  # as they normally are owned by the ddev-defined user, and here we're running
+  # as 'seluser'
+  # and do a mkcert -install so we can trust them
+  post-start:
+    - exec: "sudo chmod ugo+r /mnt/ddev-global-cache/mkcert/*"
```

Comment:
DDEV has installed the mkcert CA for use by its installed user (based on the host-side user and userid) but the chrome container is running as user `seluser`, so we need to make it readable. This sudo chmod takes place using the web container, which has sudo in it.